BarrysPeanutsAkkaFlat/src/main/java/barryspeanuts/actor/CheckOutActor.java

```diff
@@ -0,0 +1,50 @@
+package barryspeanuts.actor;
+
+import akka.actor.typed.Behavior;
+import akka.actor.typed.javadsl.AbstractBehavior;
+import akka.actor.typed.javadsl.ActorContext;
+import akka.actor.typed.javadsl.Behaviors;
+import akka.actor.typed.javadsl.Receive;
+import java.util.UUID;
+
+public class CheckOutActor extends AbstractBehavior<Object> {
+  private CheckOutActor(ActorContext<Object> context) {
+    super(context);
+  }
+
+  public static Behavior<Object> create() {
+    return Behaviors.setup(CheckOutActor::new);
+  }
+
+  @Override
+  public Receive<Object> createReceive() {
+    return newReceiveBuilder().onMessage(StartCheckout.class, this::handleStartCheckout).build();
+  }
+
+  private Behavior<Object> handleStartCheckout(StartCheckout msg) {
+    getContext()
+        .getLog()
+        .info(
+            "TODO: Getting purchase state for PurchaseId {} from data store and denote that checkout has "
+                + "started or relying upon the Akka Event Sourcing framework.",
+            msg.purchaseId);
+
+    getContext().getLog().info("Starting checkout for PurchaseId {}.", msg.purchaseId);
```

Comment:
Sometimes you use getters on these inner classes for these logs, sometimes you use fields (would suggest getters always)

Reply:
addressed